### DISCUSSION TITLE
config_relations toolbarButtons

### DISCUSSION THREAD

**Discussion by juanma0204** [wintercms/winter#519] [Q&A]
Hi, I have a questions about that relations toolbarButtons.
Actually I have in my config_relations.yaml a pivot and toolbarButtons: add|remove, but the actions add and remove execute automatic on the database add or remove the relation.
Is there some way the action doesn't execute until the user execute the button save or cancel?

Thanks you.

**Comment by bennothommo** [comment]
@juanma0204 sorry, you might have to re-word the question. I'm not quite sure what you are asking?

**Comment by juanma0204** [comment]
Yes, of course.
I will attach images to explain more.
I have a relationship in my config_relations.
Image 1 is my relationship.
The second image is my config_relation in the tab view and the last image is my view relationship in the tab.
Until then, no problem.
My problem is in the actions add|delete in relationships.
The add or delete actions automatically remove record relationships from the database.
My question is, actions can't save the record in memory until the user saves or cancels the edit?
I hope this time you understand me, sorry. 

<img width="577" alt="Captura de Pantalla 2022-04-04 a la(s) 08 38 44" src="https://user-images.githubusercontent.com/14204395/161545333-85d21117-cfd5-497b-962f-53e79874df12.png">
<img width="669" alt="Captura de Pantalla 2022-04-04 a la(s) 08 29 56" src="https://user-images.githubusercontent.com/14204395/161543989-06a3674c-2c26-4f60-af34-40ab5f2c9efe.png">
<img width="1142" alt="Captura de Pantalla 2022-04-04 a la(s) 08 33 31" src="https://user-images.githubusercontent.com/14204395/161544435-78f7454f-21e4-4c28-93be-66c9e70d0cab.png">

**Reply by fgschafer** [reply]
If I didn't missunderstood your question, I think you have to turn off defered binding for that relation manager.  There's an option for that.

**Comment by juanma0204** [comment]
Hi @fgschafer. Thank you. I'll try disabling deferred bingeing and see what happens
